src/services/member/plugins/data/controller.ts

```diff
@@ -0,0 +1,42 @@
+import { StatusCodes } from 'http-status-codes';
+
+import { FastifyPluginAsync } from 'fastify';
+
+import { buildRepositories } from '../../../../utils/repositories';
+import { exportMemberData } from './schemas/schemas';
+import { DataMemberService } from './service';
+
+const plugin: FastifyPluginAsync = async (fastify) => {
+  const {
+    files: { service: fileService },
+    mailer,
+    db,
+  } = fastify;
+  const dataMemberService = new DataMemberService();
+
+  // download all related data to the given user
+  fastify.post(
+    '/',
```

Comment:
no path?

Reply:
No because the plugin prefix path is `/export-data` so I think it is not necessary. What do you think ? Should I add a path like `request` to have `POST /members/export-data/request` to request an export ?